templates/admin/order/refund/content/sections/form/items/table/body/shipment.html.twig

```diff
@@ -19,10 +19,10 @@
         </td>
         <td class="aligned collapsing partial-refund">
 
-            <div class="ui labeled input">
-                <div class="ui label">{{ order.currencyCode|sylius_currency_symbol }}</div>
+            <div class="input-group">
+                <div class="input-group-text">{{ order.currencyCode|sylius_currency_symbol }}</div>
                 {% set inputName = "sylius_refund_shipments["~shipping_adjustment.id~"][amount]" %}
```

Comment:
Use snake_case here

Reply:
This is the template that will be refactored in the next task, so I left it without changes